src/core/stats.cc

```diff
@@ -1463,6 +1482,25 @@ Column Stats::get_stat_as_column(Stat stat) {
 }
 
 
+//------------------------------------------------------------------------------
+// Date and Time stats
+//------------------------------------------------------------------------------
+
+double TimeStats::sum (bool* isvalid) { return _invalid<double>(isvalid); }
+double TimeStats::stdev (bool* isvalid) { return _invalid<double>(isvalid);}
+double DateStats::sum (bool* isvalid) { return _invalid<double>(isvalid); }
+double DateStats::stdev (bool* isvalid) { return _invalid<double>(isvalid); }
+
+
+double DateStats::mean (bool* isvalid) {
+  if (!is_computed(Stat::Mean)) {
+    compute_moments12();
+    // Conversion from days to nanoseconds for further casting to `time64`
+    _mean *= 86400000000000;
```

Comment:
Replace this magic number with a named constant: `constexpr int64_t NANOSECONDS_IN_DAY = 24 * 3600 * 1000000000`.

Reply:
Don't you have this number already somewhere in the code? For example, when casting `date32` to `time64` or something similar.

Reply:
Yes, but it's not a globally-defined constant.

Reply:
I can see it now. Also not sure why this [one](https://github.com/h2oai/datatable/blob/main/src/core/types/type_time.cc#L87) is called `SECONDS` and not `NANOSECONDS`.

Reply:
We have quite a lot of these actually: https://github.com/h2oai/datatable/blob/main/src/core/python/date.cc#L122 At some point may be we should create something like `time_consts.h` with all these time conversion constants.